source/features/revert-file.tsx

```diff
@@ -54,12 +53,11 @@ async function deleteFile(menuItem: Element): Promise<void> {
 	await postForm(form!);
 }
 
-async function commitFileContent(menuItem: Element, content: string): Promise<void> {
+async function commitFileContent(menuItem: Element, content: string, filePath: string): Promise<void> {
 	let {pathname} = menuItem.previousElementSibling as HTMLAnchorElement;
 	// Check if file was deleted by PR
 	if (menuItem.closest('[data-file-deleted="true"]')) {
 		menuItem.textContent = 'Undeleting…';
-		const filePath = pathname.split('/')[5]; // The URL was something like /$user/$repo/blob/$startingCommit/$path
```

Comment:
Why not just change this to

```ts
const filePath = menuItem.closest<HTMLDivElement>('[data-path]')!.dataset.path!;
```

Reply:
You have it already on line 91. its used 2 times, why not use the same var again?

Reply:
What I’m suggesting is that you’re creating the variable in another function and then passing it around and just using it here. Right? So why not avoid that?

Reply:
Oh I noticed it’s used in 2 places. Looks good.